dev/bench/bench.sh

```diff
@@ -327,6 +327,7 @@ create_opam() {
 
     # Pinning for packages that are not in a repository
     opam pin add -ynq coq-perennial.dev git+https://github.com/mit-pdos/perennial#coq/tested
+    opam pin add -ynq coq-category-theory.dev git+https://github.com/jwiegley/category-theory#master
```

Comment:
Let's not have more pins. add it to opam-coq-archive instead.

Reply:
It feels weird to add somebody else's package to the opam archive, even if it is just in extra-dev.